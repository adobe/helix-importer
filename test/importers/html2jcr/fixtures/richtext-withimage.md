
+-------------------------------------------------------------------------+
| richtext                                                                |
+=========================================================================+
| Text with link [CTA 1](htttps://www.adobe.com) becomes richtext as well |
| Text with link [CTA 2](htttps://www.google.com)  as well                |
+-------------------------------------------------------------------------+

+-------------------------------------------------------------------------+
| richtext                                                                |
+=========================================================================+
| [CTA 1](htttps://www.adobe.com) becomes richtext as well                |
| [CTA 2](htttps://www.google.com)  as well                               |
+-------------------------------------------------------------------------+

+-------------------------------------------------------------------------+
| richtext                                                                |
+=========================================================================+
| Just plain text becomes richtext as well                                |
+-------------------------------------------------------------------------+

+-------------------------------------------------------------------------+
| richtext                                                                |
+=========================================================================+
| ```                                                                     |
| {                                                                       |
|   "type": "pwa-node",                                                   |
|   "name": "Action:csvimportmagento-0.0.1/push-product",                 |
|   "runtimeArgs": [                                                      |
|     "csvimportmagento-0.0.1/push-product",                              |
|     "${workspaceFolder}/actions/push-product/index.js",                 |
|     "-v",                                                               |
|     "—disable-concurrency",                                             |
|     "—kind",                                                            |
|     "nodejs:12"                                                         |
|   ]                                                                     |
| },                                                                      |
| ...                                                                     |
|  "compounds": [                                                         |
|      {                                                                  |
|        "name": "Actions",                                               |
|        "configurations": [                                              |
|          ......                                                         |
|          "Action:csvimportmagento-0.0.1/push-product"                   |
|        ]                                                                |
|      },                                                                 |
|      {                                                                  |
|        "name": "WebAndActions",                                         |
|        "configurations": [                                              |
|          ......                                                         |
|          "Action:csvimportmagento-0.0.1/push-product"                   |
|        ]                                                                |
|      }                                                                  |
|    ]                                                                    |
| ```                                                                     |
+-------------------------------------------------------------------------+
