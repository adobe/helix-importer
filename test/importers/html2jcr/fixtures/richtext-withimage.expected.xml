<?xml version="1.0" encoding="UTF-8"?>
<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="cq:Page">
    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" sling:resourceType="core/franklin/components/page/v1/page">
        <root jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/root/v1/root">
            <section sling:resourceType="core/franklin/components/section/v1/section" jcr:primaryType="nt:unstructured">
                <block sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="richtext-withimage" myText="&lt;a href=&quot;htttps://www.adobe.com&quot;>CTA 1&lt;/a>&lt;p> becomes richtext as well &lt;/p>&lt;a href=&quot;htttps://www.google.com&quot;>CTA 2&lt;/a>&lt;p> as well&lt;/p>" model="richtext-withimage"/>
                <block_0 sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="richtext-withimage" myText="&lt;p>Text with link &lt;/p>&lt;a href=&quot;htttps://www.adobe.com&quot;>CTA 1&lt;/a>&lt;p> becomes richtext as well Text with link &lt;/p>&lt;a href=&quot;htttps://www.google.com&quot;>CTA 2&lt;/a>&lt;p> as well&lt;/p>" image="image.png" imageAlt="Decorative double Helix" model="richtext-withimage"/>
                <block_1 sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="richtext-withimage" myText="&lt;p>Text with link &lt;/p>&lt;a href=&quot;htttps://www.adobe.com&quot;>CTA 1&lt;/a>&lt;p> becomes richtext as well Text with link &lt;/p>&lt;a href=&quot;htttps://www.google.com&quot;>CTA 2&lt;/a>&lt;p> as well&lt;/p>" model="richtext-withimage"/>
            </section>
        </root>
    </jcr:content>
</jcr:root>